Dockerfile

```diff
@@ -57,10 +58,10 @@ RUN set -x \
     ' \
     && apt-get update \
     && apt-get install -y --no-install-recommends $buildDeps \
-    && rm -rf /var/lib/apt/lists/* \
-    \
+    && rm -rf /var/lib/apt/lists/*
+
 # Build HAProxy
-    && wget -O haproxy.tar.gz "https://www.haproxy.org/download/$HAPROXY_MAJOR/src/haproxy-$HAPROXY_VERSION.tar.gz" \
+RUN wget -O haproxy.tar.gz "https://www.haproxy.org/download/$HAPROXY_MAJOR/src/haproxy-$HAPROXY_VERSION.tar.gz" \
```

Comment:
Please revert these changes so that it is one large RUN statement - if you would like to refactor the Dockerfile, it can be done in a separate PR.